kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/util/PppAuthSecrets.java

```diff
@@ -234,51 +231,36 @@ public void removeEntry(String type, String value) throws Exception {
         if (type.compareTo("provider") == 0) {
```

Comment:
can you please reverse the comparison ``"a string constant".equals(anotherString)``